.github/workflows/convention-develocity-gradle-plugin-verification.yml

```diff
@@ -0,0 +1,64 @@
+name: Verify Convention Develocity Gradle Plugin
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+  workflow_dispatch:
+
+jobs:
+  build:
+    name: Build Convention Develocity Gradle Plugin
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Set up JDK 8
+        uses: actions/setup-java@v4
+        with:
+          java-version: '8'
+          distribution: 'temurin'
+      - name: Set up Gradle
+        uses: gradle/actions/setup-gradle@v3
+      - name: Build with Gradle
+        working-directory: convention-develocity-gradle-plugin/plugins/gradle-5-or-newer
+        run: ./gradlew build publishToMavenLocal
+      - name: Upload plugin
+        uses: actions/upload-artifact@v4
+        with:
+          name: convention-develocity-gradle-plugin
+          path: ~/.m2/repository/com/myorg
+
+  verification:
+    name: Verify Example Build for Gradle ${{ format(matrix.gradle-version, ' ') }}
+    needs: [build]
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        gradle-version: ['5', '6{0}and{0}later'] # Gradle 4 and earlier examples are not tested as they would not succeed
```

Comment:
Because 3 and earlier does not apply CCUD, so it's not possible to override the server URL from the command line. See [here](https://github.com/gradle/develocity-build-config-samples/actions/runs/9116910759/job/25066613181#step:6:40).

And because 4 does not support publishing to authenticated servers. See [here](https://github.com/gradle/develocity-build-config-samples/actions/runs/9116910759/job/25066613424#step:6:43).

Reply:
I would recommend just leaving this useful info as a comment in the action somewhere

Reply:
Done in db7b3d1cd808c78584ea0eda802e95c5f8901d23